tests/modules/gatk4/mutect2/main.nf

```diff
@@ -100,10 +102,10 @@ workflow test_gatk4_mutect2_mitochondria {
     run_single = false
     run_pon = false
     run_mito = true
-    interval_label = 'chr22'
-    fasta = file(params.test_data['homo_sapiens']['genome']['genome_fasta'], checkIfExists: true)
-    fai = file(params.test_data['homo_sapiens']['genome']['genome_fasta_fai'], checkIfExists: true)
-    dict = file(params.test_data['homo_sapiens']['genome']['genome_dict'], checkIfExists: true)
+    interval_label = 'chr21'
+    fasta = file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true)
+    fai = file(params.test_data['homo_sapiens']['genome']['genome_21_fasta_fai'], checkIfExists: true)
+    dict = file(params.test_data['homo_sapiens']['genome']['genome_21_dict'], checkIfExists: true)
```

Comment:
change lines 96 and 97 to: 
[ file(params.test_data['homo_sapiens']['illumina']['mitochon_standin_recalibrated_sorted_bam'], checkIfExists: true)], 
and 
[ file(params.test_data['homo_sapiens']['illumina']['mitochon_standin_recalibrated_sorted_bam_bai'], checkIfExists: true)],
(apologies, wont let me comment on unchanged code lines)
Keep interval_label as 'chr22' and use the original genome_fasta , fai and dict files. This should shorten the mitochondria test immensely, I suspect this test is why the mutect2 tests are taking forever, the mitochon_standin bam file is far shorter so I kept it in to use here instead

Reply:
ah great, thank you :)